runtime/doc/various.txt

```diff
@@ -211,6 +211,11 @@ g8			Print the hex values of the bytes used in the
 :te[rminal][!]		Open an interactive shell in a |terminal-emulator|.
 			Use the shell specified in 'shell'.
 
+			The new terminal buffer takes over the current split
```

Comment:
No such thing as a "split window". Just "window". :] And maybe the whole sentence can be left away:

> Creating the terminal buffer fails when changes have been made to the current buffer, unless 'hidden' is set.